src/toil/wdl/wdl_types.py

```diff
@@ -210,7 +211,7 @@ class WDLFile:
     """
     Represents a WDL File.
     """
-    def __init__(self, file_path: str, file_name: Optional[str] = None, imported: bool = False):
+    def __init__(self, file_path: FileID, file_name: Optional[str] = None, imported: bool = False):
```

Comment:
If you need a `FileID` to make a `WDLFile`, how is `size()` going to run correctly when passes a `str` that is not a `FileID`? Won;t it end up passing it to the `WDLFile` constructor?

Reply:
Yeah, looking at it again I think it should probably stay as `str` actually. I was just a bit confused from this error mypy was throwing `src/toil/wdl/wdl_functions.py:525: error: Item "str" of "Union[Any, str]" has no attribute "size"` but I think it was because Union checks through all possible types each time. I think we also might need to put `type: ignore` here for same reason as above.